### DISCUSSION TITLE
Mac M1 - How to install php@7.3 on M1 by homebrew

### DISCUSSION THREAD

**Discussion by duongductrong** [Homebrew/discussions#1806] [Everyday usage]
I planning to buy mac pro m1 but i still consider. I researched on homebrew and found that not support php@7.3 on m1. My projects work use php version 7.3, 7.4 and more

**Comment by SMillerDev** [comment]
Run it in Rosetta mode and it should be fine. It just doesn't run natively.

**Reply by duongductrong** [reply]
Yup, thank your reply, so helpful.
